Docs/source/usage/parameters.rst

```diff
@@ -1115,8 +1115,16 @@ Laser initialization
     using ``my_constants``. For a two-dimensional simulation, similar to the B-field,
     it is assumed that the first and second dimensions are `x` and `z`, respectively,
     and the value of the `Ey` component is set to zero.
-    The current implementation of the parser for B-field on particles
+    The current implementation of the parser for E-field on particles
     is applied in cartesian co-ordinates as a function of (x,y,z) even for RZ.
+    To apply a series of plasma lenses, use the option ``repeated_plasma_lens``. This
```

Comment:
Can you please add this in an inputs case to have it covered by CI ? :)

Reply:
Thanks for your comments and fixes! I have a test case but I want to work out how to test the lens and not just have a benchmark file.

Reply:
I added a CI test case.